### PR TITLE
Donators can now select a snail-shell backpack as their special item!

### PR DESCRIPTION
**Pull request by Nickvr628**
Snail-Shell backpacks are now in game. They are identical in function to normal backpacks, except they are heavy shells so they apply slowdown like a dufflebag does. Snail people when.

image

image

:cl:  
rscadd: Donators now have a new option for their special item, the Snail Shell Backpack.
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
wow as a donator this is a great idea, speedmerge when

**Comment by optimumtact** [comment]
pay2win feature

**Comment by nichlas0010** [comment]
I hate that I don't hate this.

**Comment by Nickvr628** [comment]
Wow I meaningfully changed how donor items are spawned. Thanks monster

File: yogstation/code/modules/jobs/job_types/_job.dm [hunk 1]
```diff
@@ -17,13 +17,13 @@
...
 			hat.forceMove(BP)
-			
+
 	if(C.prefs.donor_item)
 		var/obj/item/storage/backpack/BP = locate(/obj/item/storage/backpack) in H.GetAllContents()
 		if(BP)
 			var/type = donor_start_tools[C.prefs.donor_item]
 			var/obj/item = new type()
-			item.forceMove(BP)
+			item.forceMove(put_in_hands)
```
Comment (monster860) [inline yogstation/code/modules/jobs/job_types/_job.dm]: Nick

Do you even

do you even know how to code

what

my god damn it

fucking

```suggestion
			H.put_in_hands(item)
```

File: yogstation/code/game/objects/items/storage/backpack.dm [hunk 2]
```diff
@@ -174,3 +174,22 @@
 /obj/item/disabled_boh/build_worn_icon(var/state = "", var/default_layer = 0, var/default_icon_file = null, var/isinhands = FALSE, var/femaleuniform = NO_FEMALE_UNIFORM)
 	state = "brokenpack"
 	return ..()
+
+//Nick's snail shit
+
+/obj/item/storage/backpack/snail
+	name = "snail shell"
```
Comment (Kmc2000) [inline yogstation/code/game/objects/items/storage/backpack.dm:181]: Capitals

File: yogstation/code/game/objects/items/storage/backpack.dm [hunk 3]
```diff
@@ -174,3 +174,22 @@
...
+	desc = "You wonder who this came from."
```
Comment (Kmc2000) [inline yogstation/code/game/objects/items/storage/backpack.dm:182]: You wonder from whom this came?

File: yogstation/code/game/objects/items/storage/backpack.dm [hunk 4]
```diff
@@ -174,3 +174,22 @@
...
+	icon = 'yogstation/icons/obj/storage.dmi'
+	alternate_worn_icon = 'yogstation/icons/mob/back.dmi'
+	item_state = "snail_green"
+	icon_state = "snail_green"
+	slowdown = 1
+
+
+/obj/item/storage/backpack/snail/green
+	name = "green shell backpack"
+	desc = "A emerald-green snail shell converted into a backpack. Still smells of salt"
```
Comment (Kmc2000) [inline yogstation/code/game/objects/items/storage/backpack.dm]: An?

File: yogstation/code/modules/clothing/donator.dm [hunk 5]
```diff
@@ -80,4 +80,4 @@
 	name = "clown hardsuit"
 	desc = "A clown hardsuit. The joke being that it is anything but."
 	icon_state = "hardsuit_clown"
-	item_state = "hardsuit_clown"
```
Comment (Kmc2000) [inline yogstation/code/modules/clothing/donator.dm:83]: I am the ghost of weird endlines

File: yogstation/code/modules/jobs/job_types/_job.dm (hunk 1 above)
Comment (Nickvr628) [reply yogstation/code/modules/jobs/job_types/_job.dm]: REEEEEEEEEEEEEEEEEEEEEEEEEEE

**Comment by Kmc2000** [comment]
One other thing run this sprite by par because it's really jarring.

File: yogstation/code/game/objects/items/storage/backpack.dm (hunk 3 above)
Comment (Nickvr628) [reply yogstation/code/game/objects/items/storage/backpack.dm]: yes

**Pull request merged**